consensus/polybft/extra.go

```diff
@@ -32,6 +30,7 @@ type Extra struct {
 	Parent     *Signature
 	Committed  *Signature
 	Checkpoint *CheckpointData
+	Round      uint64 // round in which the block was sealed
```

Comment:
We already have `BlockRound` in the `CheckpointData`. You should use that one instead. Also, this newly introduced Round isn't set anywhere.

Reply:
not sure I understand, this is from rfc-21, also do you think this is not used anywhere https://github.com/0xPolygon/polygon-edge/blob/EVM-154_proposer_selection/consensus/polybft/extra.go#L83

Reply:
Not used, but set. I only see it set in code when unmarshallng the `Extra`.

Reply:
ok, it was already moved to checkpoint data in the extra field so redundant field is removed.